.github/workflows/ci.yml

```
@@ -112,91 +88,48 @@ jobs:
         id: run-unit-tests
         if: success() || failure() && steps.build-platform.outcome == 'success'
         run: npm test --workspaces -- --browsers SafariNative
-    
-      - name: Create test report
-        uses: dorny/test-reporter@v1
-        if: success() || failure() && steps.run-unit-tests.outcome == 'success'
-        with:
-          name: Unit Test Results - mac
-          path: platform/reports/TESTS*.xml
-          reporter: java-junit
-          fail-on-error: true
-
-      - name: Setup .NET Core - for coverage report # Required to execute ReportGenerator
-        uses: actions/setup-dotnet@v4
-        if: success() || failure() && steps.run-unit-tests.outcome == 'success'
-        with:
-            dotnet-version: 8.x
-            dotnet-quality: 'ga'
 
-      - name: Create coverage md report 
-        uses: danielpalme/ReportGenerator-GitHub-Action@5.2.1
-        if: success() || failure() && steps.run-unit-tests.outcome == 'success'
+      - name: Upload Test Results
+        uses: actions/upload-artifact@v4  # upload test results  
+        if: ${{ !cancelled() }}           # run this step even if previous step failed
         with:
-            reports: ./platform/reports/coverage-report.xml
-            targetdir: ./platform/reports/
-            reporttypes: MarkdownSummaryGithub
-      
-      - name: Output coverage to summary
-        if: success() || failure() && steps.run-unit-tests.outcome == 'success'
-        run: cat ./platform/reports/SummaryGithub.md >> $GITHUB_STEP_SUMMARY
-
+          name: test-results-mac
+          path: "platform/reports/*.xml"
 
   build-win: 
-    runs-on: windows-latest
-    env:
+     runs-on: windows-latest
```

This may need to change as below. See my comments in #234 
```suggestion
     runs-on: windows-2025
```